script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,57 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="xbmc.addon" version="17.0.0" />
+    <import addon="xbmc.gui" version="5.14.0" />
+  </requires>
+  <extension library="service.py" point="xbmc.service" />
+  <extension library="resources/lib" point="xbmc.python.module" />
```

Comment:
something's wrong here, you're defining 2 modules. Maybe the resources one is not needed. Can you clarify?

Reply:
Removed resources/lib